app/models/shipwire/order_decorator.rb

```diff
@@ -0,0 +1,6 @@
+# Fix for retrieve order from Shipwire gem
+Shipwire::Orders.class_eval do
```

Comment:
because was only a fix, and i don't need to use `super` method. However I changed the class_eval in prepend